andela/hc-rogue-one
# Ft 156577853 Add Scheduling of standard tasks

#### What does this PR do?
Allows a logged in user to be able to schedule standard tasks
#### Description of Task to be completed?
Allows a user to schedule mailing tasks, by scheduling a task to send out mail every after a given interval. The user provides the emails of recipients, the message to be passed on and intervals after which mail should be sent out. 
#### How should this be manually tested?
You should have  REDIS already installed on the computer.
1.Install the project and install the requirements.txt
2.Run the commands below
```
Open terminal and run

cd into hc-rogue-one and we start redis server
$ start redis-server

Open new terminal

cd into hc-rogue-one and we start celery
$ celery worker -A hc -l info

Open new terminal 

cd nto hc-rogue-one and we start Beat
$ celery -A hc beat
```
#### What are the relevant pivotal tracker stories?
#156577853
## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
